roles/pulp_api/templates/pulpcore-api.service.j2

```diff
@@ -2,6 +2,10 @@
 Description=Pulp WSGI Server
 After=network-online.target
 Wants=network-online.target
```

Comment:
I wonder if we should still have these existing lines (as well as the WantedBy) in each of the regular service files.

Reply:
to be honest, I have no idea, systemd is a mystery for me